politeiawww/api/cms/v1/v1.go

```diff
@@ -22,6 +24,9 @@ const (
 	RouteInvoiceDetails      = "/invoices/{token:[A-z0-9]{64}}"
 	RouteSetInvoiceStatus    = "/invoices/{token:[A-z0-9]{64}}/status"
 	RouteUserInvoices        = "/user/invoices"
+	RouteNewDCC              = "/dcc/new"
+	RouteDCCDetails          = "/dcc/{token:[A-z0-9]{64}}"
+	RouteGetDCCs             = "/dcc/status"
```

Comment:
Shouldn't this route just be `/dcc` since status is an optional filtering param and there might be others added in the future?